packages/core/core/src/utils/transform-content-types-to-models.ts

```diff
@@ -263,6 +263,17 @@ export const transformContentTypesToModels = (contentTypes: LoadedContentTypeMod
       },
     };
 
+    // Add indexes to model
+    if (contentType.modelType === 'contentType') {
+      model.indexes = [
+        {
+          name: identifiers.getUniqueIndexName([contentType.collectionName, 'document_id']),
+          // Filter attributes that are not in the schema
+          columns: ['document_id', 'locale', 'published_at'].filter((n) => model.attributes[n]),
```

Comment:
still have to look why this is an issue, audit logs fails if not filtering locale from this list